apps/src/lab2/hooks/useFileUploader.tsx

```diff
@@ -100,8 +100,9 @@ export const useFileUploader = ({
           name: file.name,
           type: file.type,
         });
+        const [, fileType] = file.name.split('.');
         errorCallback(
-          codebridgeI18n.invalidFileUpload({fileType: file.type}),
+          codebridgeI18n.invalidFileType({fileType: file.type || fileType}),
```

Comment:
There are files with `type` not assigned. Example: .hex files. This extracts the file extension if `type` is not available.